source/_posts/2025-03-05-release-20253.markdown

```diff
@@ -0,0 +1,563 @@
+You can get the diagnostics from configuration entry or the device and open an
+issue or submit a PR with the generated diagnostics file on the
+[`aiohomeconnect` repository](https://github.com/MartinHjelmare/aiohomeconnect)
+to suggest the missing program(s).
+
+([@Diegorro98] - [#136116]) ([documentation](/integrations/home_connect))
+
+[@Diegorro98]: https://github.com/Diegorro98
+[#136116]: https://github.com/home-assistant/core/pull/136116
+
+---
+
+Using program and option keys that aren't at the API documentation, will no
+longer work. To be able to use them, open an issue or submit a pull request
+with the required keys at [MartinHjelmare/aiohomeconnect](https://github.com/MartinHjelmare/aiohomeconnect).
+
+([@Diegorro98] - [#137027]) ([documentation](/integrations/home_connect))
+
+[@Diegorro98]: https://github.com/Diegorro98
+[#137027]: https://github.com/home-assistant/core/pull/137027
+
+---
+
+Multiple Home Connect config entries will not be allowed to be configured. For
+regular users where all appliances in the home are connected to the same Home
+Connect account there should not be any reason to have more than one config
+entry since all appliances in the home will be connected to the same account
+and config entry.
+
+([@Diegorro98] - [#137088]) ([documentation](/integrations/home_connect))
+
+[@Diegorro98]: https://github.com/Diegorro98
+[#137088]: https://github.com/home-assistant/core/pull/137088
+
+{% enddetails %}
+
+{% details "MQTT" %}
+
+ Legacy `color_mode` support for MQTT JSON lights has been removed. The support
+ was deprecated in HA Core 2024.4.
+
+Users with a YAML setup have been instructed to update there configuration in
+`configuration.yaml` via repair issue flow. Deprecation issues with MQTT JSON
+lights that were set up through discovery were communicated via warnings in
+the logs. Use of the removed parameters in discovery wil no break
+the configuration, but will still log warning.
+
+([@jbouwh] - [#136996]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#136996]: https://github.com/home-assistant/core/pull/136996
+
+{% enddetails %}
+
+{% details "Proximity" %}
+
+The distance is now measured to the edge of the monitored zones (it takes the
+radius into account), instead of to the center of the zone as before. This leads
+to slightly different expected distance values, which you may have to adjust
+accordingly in your existing automations.
+
+([@Spcemarine] - [#138819]) ([documentation](/integrations/proximity))
+
+[@Spcemarine]: https://github.com/Spcemarine
+[#138819]: https://github.com/home-assistant/core/pull/138819
+
+{% enddetails %}
+
+{% details "SmartThings" %}
+
+The energy and power sensors have been removed for every device that has a
+switch, since they never really worked, because there were no energy or power
+capabilities available to get data from. If you have the capabilities,
+it will still work for you.
```

Comment:
```suggestion
The energy and power sensors have been removed for every device that has a
switch. They rarely worked because there were no energy or power
capabilities available to get data from. If you do have the capabilities,
it will still work for you.
```